### PR TITLE
Cosmetic changes

### PR DESCRIPTION
**Pull request by WildFireFlum**
Minor cosmetic change.
This should be merged after https://github.com/vmware/concord-bft/pull/2403.

### CODE REVIEW

File: tests/apollo/test_skvbc_time_service.py [hunk 1]
```diff
@@ -303,7 +303,7 @@ async def test_restart_non_primary_replica(self, bft_network, tracker):
 
     @with_trio
     @with_bft_network(start_replica_cmd,
-            selected_configs=lambda n, f, c: c == 0 and n >= 6, rotate_keys=True)
+            selected_configs=lambda n, f, c: c == 0 and n >= 6, rotate_keys=True, retries=4)
```
Comment (situ-s) [inline tests/apollo/test_skvbc_time_service.py:306]: Just wanted to know, why max_retries has been set as specific number 4?

**Review by situ-s — APPROVED** [review]
Thanks, LGTM!!

**Comment by WildFireFlum** [comment]
@yuliasherman Should I leave the retries as-is?

**Comment by yuliasherman** [comment]
> @yuliasherman Should I leave the retries as-is?

If there wasn't any specific reason for doing that, please revert.

**Pull request merged**